src/snowflake/snowpark/modin/pandas/series_utils.py

```diff
@@ -1428,90 +1430,82 @@ def tz(self) -> "tzinfo | None":
 
     @property
     def freq(self):
-        return self._query_compiler.dt_freq().to_pandas().squeeze()
+        return self._query_compiler.dt_property("freq").to_pandas().squeeze()
 
     def to_period(self, *args, **kwargs):
-        return Series(query_compiler=self._query_compiler.dt_to_period(*args, **kwargs))
+        return Series(query_compiler=self._query_compiler.dt_property("to_period"))
```

Comment:
This is a method not a property. I think it's better to add a corresponding method in `SnowflakeQueryCompiler` and raise a NotImplementedError there.

Reply:
Ah I missed that. Let me update this